packages/tinyfugue/SPECS/tinyfugue.spec

```diff
@@ -0,0 +1,153 @@
+%global packageversion 50b8
+
+Name:    tinyfugue
+Version: 5.0
+Release: 0.32.b8%{?dist}
+Summary: A MU* client
+License: GPLv2+
+URL:     http://tinyfugue.sourceforge.net/
+Source:  http://downloads.sourceforge.net/tinyfugue/tf-%{packageversion}.tar.gz
+Patch0:  tf-50b7.build.patch
+Patch1:  tf-50b8.x86_64.patch
+Patch100: tinyfugue508.irix.patch
+# https://sourceforge.net/tracker/?func=detail&aid=3486514&group_id=186112&atid=915972
+Patch2:  tf-50b8.pcre.patch
+BuildRequires: coreutils
+BuildRequires: gcc
+BuildRequires: make
+BuildRequires: ncurses-devel
+#BuildRequires: pcre-devel
```

Comment:
I think this should be active? (though honestly I'm browsing, not checking by compiling).

Reply:
Fixed